source/training/session4.md

```diff
@@ -0,0 +1,31 @@
+# Using the web API​
+
+<div class="present"><div class="reveal"><div class="slides">
+
+  <section class="has-dark-background" data-background="#240b35" data-background-image="/_static/img/openfisca-bg.svg" data-background-position="230% 50%" data-background-size="auto 120%">
+    <h2>Using the web API​​</h2>
+    <p>
+      <small>(Rules as Code + OpenFisca) training series, session four.</small>
+    </p>
+    <p><a href="https://openfisca.org"><img src="/_static/img/openfisca.svg" alt="OpenFisca Logo" style="height: 70px; margin: 17rem auto 0 auto; background: transparent;"  ></a></p>
+    <p><small><span style="font-size:0.7em;">created by <a href="https://hamish.dev">Hamish Fraser</a></span></small></p>
+  </section>
```

Comment:
I find it risky to use inline CSS and HTML like this. This will require adjustments when the layout is changed. Couldn't we use Markdown and CSS, potentially creating a dedicated layout for the training sessions? 🙂